### PR TITLE
ZFIN-9356: Add measures to find empty files for pubs

### PR DESCRIPTION
**Pull request by rtaylorzfin**
Added a jenkins job for finding empty files.

Added a UI element to help make it more obvious when an empty file is uploaded:

https://github.com/user-attachments/assets/881dd776-234e-46f8-845a-ee07d72e8eb5

### CODE REVIEW

File: home/javascript/react/containers/PubFileCheck.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        const tempEmptyFiles = tempFiles.filter((file) => file.size === 0);
+        setFiles(tempFiles);
+        setEmptyFiles(tempEmptyFiles);
+    }
+
+    useEffect(() => {
+        setTimeout(() => {
+            fetchFiles();
+        });
+        // fetchFiles();
```
Comment (cmpich) [inline home/javascript/react/containers/PubFileCheck.tsx:22]: Could this be removed?

**Review by cmpich — COMMENTED** [review]
This jenkins job is executed manually? I don't see triggers or email lists.

**Review by cmpich — APPROVED** [review]
Looks good except the one comment line that probably can be removed.

File: home/javascript/react/containers/PubFileCheck.tsx (hunk 1 above)
Comment (rtaylorzfin) [reply home/javascript/react/containers/PubFileCheck.tsx]: Thanks, good catch

**Pull request merged**